sql/updates/latest-dev.sql

```diff
@@ -182,23 +182,24 @@ WHERE job_type = 'drop_chunks'
   AND job.id = c.job_id;
 
 -- migrate cagg jobs
+-- config will be updated later
 UPDATE
   _timescaledb_config.bgw_job job
 SET
-  application_name = format('%s [%s]', 'Continuous Aggregate Policy', id),
+  application_name = format('%s [%s]', 'Refresh Policy', id),
   proc_schema = '_timescaledb_internal',
-  proc_name = 'policy_continuous_aggregate',
-  config = jsonb_build_object('mat_hypertable_id', c.mat_hypertable_id),
-  hypertable_id = c.mat_hypertable_id,
+  proc_name = 'policy_refresh_continuous_aggregate',
+  config = NULL,
```

Comment:
why are you not setting config? hypertable_id in the jobs table should only be used for dependency tracking

Reply:
See post-update.sql for config setting. The timescale sql functions do not work in latest-dev.sql